model/role/db.go

```diff
@@ -0,0 +1,39 @@
+package role
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/anser/bsonutil"
+	adb "github.com/mongodb/anser/db"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+var (
+	IdKey          = bsonutil.MustHaveTag(Role{}, "Id")
+	NameKey        = bsonutil.MustHaveTag(Role{}, "Name")
+	ScopeTypeKey   = bsonutil.MustHaveTag(Role{}, "ScopeType")
+	ScopeKey       = bsonutil.MustHaveTag(Role{}, "Scope")
+	PermissionsKey = bsonutil.MustHaveTag(Role{}, "Permissions")
```

Comment:
I've always thought that it's weird that we keep the keys seperate from the model which makes it harder to figure out what you have to do when you want to add a field.

Reply:
Meh, I don't think it helps much though. I feel like the cleanest way to do this would be via reflection

Reply:
take that back.